README.md

```diff
@@ -148,6 +157,14 @@ Ensure your CometBFT P2P port is open and accessible from the outside.
 This is `26656` by default, but can be changed using the `p2p.laddr` or 
 `p2p.external_address` parameters in the `config.toml` file.
 
+#### Lightweight seed nodes
+
+To run a Seed Node with minimal resources, you have to additionally:
+- Set `pruning=everything` in `app.toml` to enable storing only current chain state
+- Set `state-sync.snapshot-interval` to `0` to disable snapshots of the state
+
+Setting those parameters can significantly reduce node's storage usage, thus improving the resource efficiency.
```

Comment:
Let's move this entirely under "Network seed nodes". No need to introduce "lightweight". All seed nodes should follow this.